src/DynamoMLDataPipeline/UploadAssetsRequestBody.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace DynamoMLDataPipeline
+{
+    // Schema and the assets for the data request body.
+    class UploadAssetsRequestBody
```

Comment:
You really should use access modifiers to make this explicit, I think default is private.

Reply:
Default is internal for classes. Added internal modifier to these classes explicitly. https://github.com/DynamoDS/Dynamo/pull/15116.